README.md

```diff
@@ -1,66 +1,89 @@
-- [📚 Cookbooks](#-cookbooks)
-- [💻 Development with Cursor](#-development-with-cursor)
+## [NEW] 🎆 Agent Reinforcement Learning
+
+Train your agents with multi-turn reinforcement learning using [Fireworks AI](https://fireworks.ai/)! Judgeval now integrates with Fireworks' Reinforcement Fine-Tuning (RFT) endpoint, supporting gpt-oss, qwen3, Kimi2, DeepSeek, and more.
+
+Judgeval provides a simple harness for integrating GRPO into any Python agent, giving builders a quick method to **try RL with minimal code changes** to their existing agents!
+
+```python
+await trainer.train(
+    agent_function=your_agent_function,
+    scorers=[RewardScorer()],  # Custom scorer you define based on task criteria
+    prompts=training_prompts,  # Tasks
+    rft_provider="fireworks"
+)
+```
+
+**That's it!** Judgeval automatically manages trajectory collection and reward tagging - your agent can learn from production data with minimal code changes. You can view and monitor training progress for free via the [Judgment Dashboard](https://app.judgmentlabs.ai/).
+
+## Judgeval Overview
+
+Judgeval is an open-source framework for agent behavior monitoring. Judgeval offers a toolkit to track and judge agent behavior in online and offline setups, enabling you to convert interaction data from production/test environments into improved agents. To get started, try running one of the notebooks below depending on your use case or diving deeper in our [docs](https://docs.judgmentlabs.ai/documentation).
+
+Our mission is to unlock the power of production data for agent development, enabling teams to improve their apps by catching real-time failures and optimizing over their users' preferences.
+
+## 📚 Cookbooks
+
+| Try Out | Notebook | Description |
+|:---------|:-----|:------------|
+| Custom Scorers | [Get Started For Free] | Build custom evaluators for your agents |
+| Online Monitoring | [Get Started For Free] | Monitor agent behavior in production |
+| RL | [Get Started For Free] | Train agents with reinforcement learning |
+
+You can access our repo of cookbooks [here](https://github.com/JudgmentLabs/judgeval-cookbook).
+
+
+## Why Judgeval?
+
+• **Custom Evaluators**: Judgeval provides simple abstractions for custom evaluators and their applications to your agents, supporting LLM-as-a-judge and code-based evaluators that connect to datasets our and metric-tracking infrastructure. [Learn more](https://docs.judgmentlabs.ai/documentation/evaluation/scorers/custom-scorers)
+
+• **Production Monitoring**: Run any custom scorer online in production. Get Slack alerts for failures and add custom hooks to address regressions before they impact users. [Learn more](https://docs.judgmentlabs.ai/documentation/performance/online-evals)
 
-## 🛠️ Installation
+• **Simple to run multi-turn RL**: Go from agent code to optimization via RL (supports multi-turn) easily without managing compute infrastructure or data pipelines. Simply plug onto your agents in production and train!
```

Comment:
Optimize your agents with multi-turn RL without managing compute infrastructure or data pipelines. Just add a couple lines of code to your agent and train!

TODO: Add link to docs once we have a section on `judgment.train`